app/Http/Requests/ValidationRequest.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace App\Http\Requests;
+
+use Illuminate\Foundation\Http\FormRequest;
+
+class ValidationRequest extends FormRequest {
+    /**
+     * Determine if the user is authorized to make this request.
+     *
+     * @return bool
+     */
+    public function authorize() {
+        return true;
+    }
+
+    /**
+     * Get the validation rules that apply to the request.
+     *
+     * @return array
+     */
+    public function rules() {
+        return [
+            'name' => 'required|max:255',
+        ];
+    }
+
+    public function messages() {
+        return [
+            'required' => trans('lang.requiredVali'),
+            'max' => trans('lang.maxVali'),
+        ];
```

Comment:
Nghiên cứu ở https://laravel.com/docs/5.5/validation#form-request-validation phần `Customizing The Error Messages` để customize messages cho đúng em nhé